tests/function.rs

```diff
@@ -67,6 +67,20 @@ fn body_change_not_detected() {
     assert!(diff.is_empty());
 }
 
+#[test]
+fn fn_arg_comma_is_removed() {
+    let diff: ApiCompatibilityDiagnostics = parse_quote! {
+        {
+            pub fn a(a: t, b: t, c: t,) {}
+        },
+        {
+            pub fn a(a: t, b: t, c: t) {}
+        },
```

Comment:
If we try "pub fn a(a: t, b: t, c: t) {}" won't it remove the "t" ? 
if not it could be nice to have a test to show it